addons/website/tests/test_ui.py

```diff
@@ -542,3 +542,33 @@ def test_website_no_dirty_page(self):
         website.menu_id.child_id[1:].unlink()
 
         self.start_tour('/', 'website_no_dirty_page', login='admin')
+
+    def test_website_no_dirty_lazy_image(self):
+        website = self.env['website'].browse(1)
+        # Set multiple langs on website (_activate_lang does not load .pot so it
+        # is perf friendly)
+        lang_fr = self.env['res.lang']._activate_lang('fr_FR')
+        lang_es = self.env['res.lang']._activate_lang('es_AR')
+        lang_zh = self.env['res.lang']._activate_lang('zh_HK')
+        lang_ar = self.env['res.lang']._activate_lang('ar_SY')
+        website.language_ids = self.env.ref('base.lang_en') + lang_fr + lang_es + lang_zh + lang_ar
```

Comment:
May be perf friendly but why enabling more than one ? 1 < 4

Reply:
> Adding 4 langs instead of one completely prevents that without much cost.

I see, maybe just add a small comment about that otherwise a future refactoring to speed up test might just remove it